src/client/utils/dom.js

```diff
@@ -277,8 +277,7 @@ export function isDomElement (el) {
         return false;
 
     // NOTE: B252941
-    return el && (typeof el === 'object' || isFirefox && typeof el === 'function') &&
-           el.nodeType !== 11 && typeof el.nodeName === 'string' && el.tagName;
+    return el && el.nodeType !== 11 && typeof el.nodeName === 'string' && el.tagName;
```

Comment:
I tried to click <object> tag in playground and using testcafe.
All works right.